app/services/finance/ecf/mentor/statement_calculator.rb

```diff
@@ -35,18 +35,20 @@ def voided_declarations
         end
 
         event_types.each do |event_type|
-          define_method "#{event_type}_fee_for_declaration" do
+          define_method "#{event_type}_fee_per_declaration" do
             fee_for_declaration(type: event_type)
           end
 
           define_method "additions_for_#{event_type}" do
-            hash = {}.merge(*output_calculator.output_breakdown)
-            hash[:"#{event_type}_additions"] * output_calculator.fee_for_declaration(type: event_type)
+            output_calculator.output_breakdown.sum do |hash|
```

Comment:
do you mind explaining why we needed those changes as it's not explained in the commit message please?

Reply:
Just for easy understanding... I've improved the commit message.